minvws/nl-kat-coordination
# Update Wappalyzer

### Changes

This update replaces the outdated and archived Wappalyzer dependency with a more current fork. It also implements a script that downloads an updated technologies file from [this](https://github.com/enthec/webappanalyzer), a project that aims to maintain Wappalyzer technologies files. For now the technology file should be updated manually from time to time, but we should work towards a plans for an automated way of doing this.

### Issue link

Closes #3037 

### QA notes

Run the Wappalyzer normalizer and see if it still produces `Software` and `SoftwareInstance` OOIs.

---

### Code Checklist







---

## Checklist for code reviewers:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---

## Checklist for QA:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.
